doc/source/rllib-optimizers.rst

```diff
@@ -0,0 +1,51 @@
+Using Policy Optimizers outside RLlib
```

Comment:
consider just renaming to Policy Optimizers

Reply:
Done